### ISSUE TITLE
Not functioning with dotnet core 5.0

### ISSUE THREAD

**Issue by CySpiegel** [alzuma/electron-demo#5]
can you please update this for dot net core 5.0. The current version does not function with current released versions of software. Current repo will not run with dotnet core 5.

**Comment by alzuma** [comment]
@CySpiegel I could try :) have you tried to adjust the code yourself?

**Comment by CySpiegel** [comment]
Yes, i have followed the website tutorial you have posted using the most up to date versions but i could not get it to work. Even checked the code against the repo and tried to run the current repo to no avail. I went to the deve page for the Electron plugin youre using and got the message to send from React to C# but it will not work in reverse.
